src/APIService.js

```diff
@@ -25,6 +26,9 @@ class APIService {
 
     this.app = express()
     this.app.use(express.json())
+
+    this.app.use(expressLogger)
+
```

Comment:
perfect as middleware, can also place below (after routes, (since all call next) before analytics) so that it is logged after response is sent, but this is such little overhead dont think it matters, plus must be non blocking anyways

Reply:
True, maybe a little bit, but it does most of the work once the request cycle is done (https://github.com/villadora/express-bunyan-logger/blob/master/index.js#L190). Usually attaching the logger is the first thing you do, and I guess I never thought about it before. I think the main benefits are that attaching at the start gives you response time, and I think may be more likely to catch errors (for example, if a middleware/handler doesn't call next for some reason)

Reply:
nice!, yeah assume it was some what optimized to not block request cycle, and yeah that makes sense 👍 , yeah we try to catch all errors to pass through, but this is more reliable still